src/modules/ticket-fields/translations/en-us.yml

```diff
@@ -0,0 +1,32 @@
+#
+# This file is used for the internal Zendesk translation system, and it is generated from the extract-strings.mjs script.
+# It contains the English strings to be translated, which are used for generating the JSON files containing the translation strings
+# for each language.
+#
+# If you are building your own theme, you can remove this file and just load the translation JSON files, or provide
+# your translations with a different method.
+#
+title: "Copenhagen Theme Ticket Fields"
+packages:
+  - "ticket-fields"
```

Comment:
I don't think it is a good idea to use `ticket-fields` as the package name and prefix for the keys, since these things are shared across Zendesk.

For `service-catalog` and `new-request-form` we did that, but we own these features and are not implemented anywhere else. Ticket fields are used everywhere, so maybe it is better to use something like `cph-theme-ticket-fields`.

Other notes:
- we are missing the `new-request-form.credit-card-digits-hint` translation. We need to move it here and use the new key in the Credit Card component
- we still need to handle translations properly. the `new-request-form` module now needs to load its own translations and the `ticket-fields` translations. Same for the `service-catalog` module. We can do it later in another PR, so we can move on with the implementation. I'll create a Jira for that
- We will need to deprecate the old strings in the `new-request-form` module. As above we can do it later

Reply:
Ok so for now I've just changed package name to `cph-theme-ticket-fields`.
I see the task that you have created so I will take it later.